app/src/main/java/com/stl/skipthelibrary/LoginActivity.java

```diff
@@ -10,15 +10,25 @@
 import android.widget.Toast;
 
 public class LoginActivity extends AppCompatActivity {
-private EditText passwordText;
-private EditText emailText;
+
+    private EditText passwordText;
+    private EditText emailText;
+    private DatabaseHelper databaseHelper;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
         emailText = findViewById(R.id.EmailEditText);
         passwordText = findViewById(R.id.PasswordEditText);
+        databaseHelper = new DatabaseHelper(this);
+
+        // user is already logged in
+        if (databaseHelper.isUserLoggedIn()){
+            DatabaseHelper databaseHelper = new DatabaseHelper(this);
```

Comment:
@tymoorej lol it looks like you removed another one but not the one I was referring to